qiskit_ibm_runtime/runtime_job.py

```diff
@@ -111,7 +111,8 @@ def __init__(
         self._params = params or {}
         self._creation_date = creation_date
         self._program_id = program_id
-        self._status = JobStatus.INITIALIZING
+        self._state = {"status": JobStatus.INITIALIZING}
```

Comment:
`self._status` does not map to the API response format directly. Why do you introduce a more complicated data structure here? Can't we leave it as before?